src/view/shell/desktop/LeftNav.tsx

```diff
@@ -485,8 +486,14 @@ function ComposeBtn() {
     return undefined
   }
 
-  const onPressCompose = async () =>
+  const onPressCompose = async () => {
     openComposer({mention: await getProfileHandle()})
+    try {
+      throw new Error('taco')
```

Comment:
leftover?

Reply:
Yep yep just here for debug and test sourcemaps, left a todo to remove later